src/main/kotlin/app/Greeter.kt

```diff
@@ -1,13 +1,15 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+
 package app
 
 import app.utils.Args
 import app.utils.EmailValidator
 
 class Greeter(args: Args) {
-    var email: String? = args.email
-    var password: String? = args.password
-    var path: String? = args.path
-    var silent: Boolean = args.silent
+    var email: String? = args.email // Sourcerer account email.
```

Comment:
I don't believe we'll be using an email as user id. We will be giving them handles just like github does.